venia-integration-tests/src/actions/offline/index.js

```diff
@@ -0,0 +1,4 @@
+export const checkServiceWorker = () => {
+    const serviceWorkerStatus = navigator?.serviceWorker?.controller?.state;
```

Comment:
This could be on one line.
`return navigator?.serviceWorker?.controller?.state;`

This seems more like a support command than a user interaction.
Also, it does not seem to be used anywhere.